internals/secrethub/acl_list_test.go

```diff
@@ -176,8 +176,8 @@ func TestACLListCommand_run(t *testing.T) {
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			// Setup
-			io := fakeui.NewIO(t)
-			tc.cmd.io = io
+			testIO := fakeui.NewIO(t)
+			tc.cmd.io = testIO
```

Comment:
Would this work?
```go
tc.cmd.io = fakeui.NewIO(t)
```

Reply:
Unfortunately, it will not. By doing what you suggested, I will no longer be able to get the output of the fake IO, since the `cmd.io` is an interface, while the fake one is a struct.